### PR TITLE
Project: Personal porto-folio Popup-window

### PR DESCRIPTION
**Pull request by Irirwanirira**
## My PR consists of :

- [ ] Added pop window with javaScript function
- [ ] Styled pop-window  i a new css file
- [ ] Modified HTML

### Thank you for your time.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -34,3 +34,130 @@ function invalidEmail(e){
...
+    },
+    {
+      title: "Multi-Post Stories ",
+      src: "images/Snapshoot-Portfolio-2.png",
+      desription:
+        "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+      technologies: ["html", "css", "javaScript"],
+      seeLive: "https://irirwanirira.github.io/My-portofolio/",
+      seeSourceLink: "https://irirwanirira.github.io/My-portofolio/",
+    },
```
Comment (codepantha) [inline script.js]: - [ ] The projects section in your HTML code is still being hardcoded. Please ensure that the projects section is created dynamically with the information from this `Project` array 👍 To do this, kindly remove all already created HTML code for projects list and add it via JavaScript

File: script.js [hunk 2]
```diff
@@ -34,3 +34,130 @@ function invalidEmail(e){
...
+              <p class="canopy">CANOPY</p>
+              <ul>
+                  <img src="images/Counter.png" alt="Dot">
+                  <li class="opt-period-pop"> Back End Dev </li>
+                  <li img src="images/Counter.png" alt="Dot"></li>
+                  <img src="images/Counter.png" alt="Dot">
+                  <li class="opt-period-pop"> 2015</li>
+  
+              </ul>
+          </div>
```
Comment (codepantha) [inline script.js]: - [ ] Well done for your work so far on implementing the popup modal. However, the modal appears at the bottom of the page and the user has to scroll to the bottom to see the popup. Apart from the fact that this is not the expected behavior, it also has obvious consequences for user experience. The modal popup is expected literally popup and appear above everything else on the page

- [ ] Please use CSS to style the modal and ensure that it appears correctly as shown in the Figma design
## Your Design
image
## Figma Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used correct GitHub Flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @Irirwanirira and @Abe1able 👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good work you have done until now! 

Please let me inform you that what the previous reviewer mentioned is not that the project's pop-up is hard coded, but that the projects section in your HTML codebase is still being hardcoded. 

Please check the following aspect of the project requirements:
image

In order to do this, you literally need to delete the project cards from the HTML file and inject them dynamically with the use of JavaScript from that array of objects.

Regarding the style of the pop-up, please do your best to style it as close as possible to the Figma design.

Your implementation | Figma Design
:------:|:------:
image | image

Please ask for another review in your Dashboard when the changes are implemented :)

Keep rocking 🚀!

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Joseph, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**